Exercises/3-by-reference.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
-const inc = null;
+const inc = function (obj) {
```

Comment:
Use short lambda syntax
```suggestion
const inc = obj =>
```